Server/src/config/envs/docker.ts

```diff
@@ -106,12 +110,18 @@ export function createDockerConfig() {
             host: '0.0.0.0',
             port: 8443,
             protocol: 'ocpp2.0.1',
-            tlsKeyFilePath:
-              '/usr/local/apps/citrineos-core/Server/src/assets/certificates/leafKey.pem',
-            tlsCertificateChainFilePath:
-              '/usr/local/apps/citrineos-core/Server/src/assets/certificates/certChain.pem',
-            rootCACertificateFilePath:
-              '/usr/local/apps/citrineos-core/Server/src/assets/certificates/rootCertificate.pem',
+            tlsKeyFilePath: path.resolve(
+              path.dirname(__filename),
```

Comment:
These changes belong in their own PR.

Reply:
not a problem 👍 I can quickly split this out into a separate PR, will adjust

Reply:
created a separate PR here - https://github.com/citrineos/citrineos-core/pull/210